### PR TITLE
Create new placeholder homepage

### PR DESCRIPTION
**Pull request by nnekaO**
Changes to the homepage for urgent care prototypes

TM-71 Create prototype: The Care Carder

### CODE REVIEW

File: app/views/index.1.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
+{% set title = 'Home' %}
```
Comment (beseven) [inline app/views/index.1.html]: Moved the original index so it can be referred back during the build. Good!

File: package-lock.json [hunk 2]
```diff
@@ -2547,11 +2547,13 @@
         },
```
Comment (beseven) [inline package-lock.json]: This is due to running on Windows - it shouldn't cause major issues.

File: app/views/index.1.html (hunk 1 above)
Comment (jbroodie) [reply app/views/index.1.html]: I'd probably just remove it? There are other ways we can revisit it.

File: app/views/index.html [hunk 3]
```diff
@@ -24,147 +24,19 @@ <h1>NHS.UK redesign project</h1>
...
-                      <tr>
-                        <td><a href="/styles/cookie-banner">Cookie banner</a></td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__design">Design</span></td>
-                        <td></td>
-                      </tr>
-                      </tbody>
-                    </table>
+                  <h2>Prototypes</h2>
+                  <ul>
+                    <li><a href="#"> Prototype 1</a></li>
```
Comment (jbroodie) [inline app/views/index.html]: Would it be better to use the names we gave the prototypes? Or add a description? I've no idea which prototype is 1, 2, or 3. But, if we as a team, have already defined and I've overlooked it somehow ignore me.

File: app/views/index.html [hunk 4]
```diff
@@ -24,147 +24,19 @@ <h1>NHS.UK redesign project</h1>
...
+                    <li><a href="#"> Prototype 3</a></li>
+                  </ul>
 
+                
                     <div class="nhsuk-reading-width">
                         <h2>Useful information</h2>
                         <ul>
-							              <li>our <a href="examples">example pages</a> for user testing and visual examples.</li>
-                            <li>the <a href="https://github.com/nhsuk/frontend.git">public GitHub repository</a> of our design code.</p></li>
+							              <li>our <a href="examples">example component pages</a> for user testing and visual examples.</li>
```
Comment (jbroodie) [inline app/views/index.html]: Do we need this?

File: package-lock.json (hunk 2 above)
Comment (jbroodie) [reply package-lock.json]: Interesting, I didn't get any of these changes.

File: app/views/index.1.html (hunk 1 above)
Comment (beseven) [reply app/views/index.1.html]: We can still have them in the repo for easy access, though.

File: app/views/index.html (hunk 3 above)
Comment (beseven) [reply app/views/index.html]: It's a placeholder and we can pick that up after.

File: app/views/index.html (hunk 4 above)
Comment (beseven) [reply app/views/index.html]: There are still examples there

File: package-lock.json (hunk 2 above)
Comment (beseven) [reply package-lock.json]: @nnekaO can you remove this, please?

**Pull request merged**